src/components/common/ContextualSaveBar.jsx

```
@@ -0,0 +1,307 @@
+            </div>
+          </div>
+        </div>
+
+        {/* Right side - Action buttons */}
+        <div className="flex items-center gap-3">
+          {/* Discard button */}
+          <button
+            onClick={onDiscard}
+            disabled={isSaving}
+            className="group relative px-5 py-2.5 bg-white/10 hover:bg-white/20 border-2 border-white/30 hover:border-white/50 rounded-lg text-white font-semibold transition-all duration-300 hover:scale-105 backdrop-blur-sm disabled:opacity-50 disabled:cursor-not-allowed"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)